src/backend/utils/add-feed.js

```diff
@@ -0,0 +1,49 @@
+require('../lib/config');
+
+const args = require('minimist')(process.argv.slice(2));
+const { logger } = require('./logger');
+const { isValidUrl } = require('./utils');
+const feedQueue = require('../feed/queue');
+
+const log = logger.child({ module: 'add-feed' });
+
+async function add() {
+  if (
+    !Object.prototype.hasOwnProperty.call(args, 'name') ||
+    !Object.prototype.hasOwnProperty.call(args, 'url')
+  ) {
+    log.error('INVALID NUMBER OF ARGUMENTS');
+    process.exit(1);
+  }
+  const { name, url } = args;
+  if (!isValidUrl(url)) {
+    log.error('INVALID URL');
+    process.exit(1);
+  }
+  const cleanName = String(name);
+  if (cleanName.match(/[.*+?^${}()|[\]\\`]/)) {
```

Comment:
What's the goal here?  Why do we need to validate names?  What if someone's name contains unicode or some such?

Reply:
I was over thinking it. Over engineered, will remove